### PR TITLE
Change check_mod_actions flag defaults.

### PR DESCRIPTION
**Pull request by jetpack**
Wait 24 hours instead of 12. Use the comment's creation time as basis for
waiting instead of bot's scoring time (should be similar, but when bot is
falling behind or when getting historical data, it can be different).

(Based on top of #25 )

### CODE REVIEW

File: perspective_reddit_bot/check_mod_actions.py [hunk 1]
```diff
@@ -30,10 +30,23 @@
...
 APPROVED_COL = 'approved'
 REMOVED_COL = 'removed'
 DELETED_COL = 'deleted'
+# score, ups, and downs refer to user votes. in practice, it appears that
+# 'downs' is always 0 and 'score' == 'ups'. (this may be different if the bot is
+# a mod.)
+SCORE_COL = 'score'
+UPS_COL = 'ups'
+DOWNS_COL = 'downs'
+SCORE_HIDDEN_COL = 'score_hidden'
```
Comment (nthain) [inline perspective_reddit_bot/check_mod_actions.py]: Add a description of score_hidden? I assume it's a boolean indicating if the score is hidden during the initial stages of a comment's life? 
https://www.reddit.com/r/redditdev/comments/1dfdpp/api_change_score_hidden_added_to_comments_and/

File: perspective_reddit_bot/log_subreddit_comments.py [hunk 2]
```diff
@@ -50,10 +50,11 @@ def comment_url(comment):
   return 'https://reddit.com' + comment.permalink
 
 
-def append_record(output_path, record):
+def append_records(output_path, records):
```
Comment (nthain) [inline perspective_reddit_bot/log_subreddit_comments.py]: Just curious, why change this if it's only being used for one record at a time at the moment?

File: perspective_reddit_bot/perspective_client.py [hunk 3]
```diff
@@ -154,7 +154,8 @@ def score_texts(self, texts, models, language, raise_on_error,
     every_5_percent = _progress_points(frac=0.05, total=len(texts))
     # pylint: disable=missing-docstring
 
-    def score_one((i, text)):
+    def score_one(itext):
```
Comment (nthain) [inline perspective_reddit_bot/perspective_client.py]: Neat!

File: perspective_reddit_bot/log_subreddit_comments.py (hunk 2 above)
Comment (jetpack) [reply perspective_reddit_bot/log_subreddit_comments.py]: it's used in the next PR

File: perspective_reddit_bot/check_mod_actions.py (hunk 1 above)
Comment (jetpack) [reply perspective_reddit_bot/check_mod_actions.py]: done

**Pull request merged**